app/components/post_header/post_header.js

```diff
@@ -152,7 +158,7 @@ export default class PostHeader extends PureComponent {
             return (
                 <TouchableWithFeedback
                     onPress={this.handleUsernamePress}
-                    style={style.displayNameContainer}
+                    style={[style.displayNameContainer, {maxWidth: showReply && isBot ? '55%' : '60%'}]}
```

Comment:
the `displayNameContainer` style already defines `maxWidth` as 60%, in order to av oid unneeded re-renders by changing the style object every time, you could define a second style like
```
displayNameContainerBotReplyWidth: {
    maxWidth:  '55%',
},
```

the do something like 
```
const displayNameStyle =  [style.displayNameContainer, showReply && isBot ? style.displayNameContainerBotReplyWidth : null];
```

and in the `TouchableWithFeedback style={displayNameStyle}` or something like that

Reply:
and definitely write some unit tests for this

Reply:
Updated